### PR TITLE
Fixed a couple of spelling mistakes

### PR DESCRIPTION
**Pull request by mage2pratik**
### Description (*)

Fixed a couple of spelling mistakes.
### Fixed Issues (if relevant)

N/A
### Manual testing scenarios (*)

N/A
### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @mage2pratik. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: lib/internal/LinLibertineFont/ChangeLog.txt [hunk 1]
```diff
@@ -952,7 +952,7 @@ Changes to version 0.5.8 regular(|) & italic(/) (20040315)
 Changes to version 0.5.7 regular(|) & italic(/) (20040315)
 -N is now 66pt wider
 -^ {Ascicircum} is now better
-- {exclamdown} is now availible
+- {exclamdown} is now available
```
Comment (orlangur) [inline lib/internal/LinLibertineFont/ChangeLog.txt]: This is third-party library but not code, so, should not cause any troubles.

**Review by orlangur — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @orlangur, thank you for the review.
[ENGCOM-3876](https://jira.corp.magento.com/browse/ENGCOM-3876) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @mage2pratik. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.1. 
Please check the release notes for final confirmation.

**Comment by magento-engcom-team** [comment]
Hi @mage2pratik. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.1. 
Please check the release notes for final confirmation.

**Pull request merged**